### PR TITLE
Setup documentation website

### PR DESCRIPTION
**Pull request by gabro**

### CODE REVIEW

File: packages/bento-design-system/src/Button/Config.ts [hunk 1]
```diff
@@ -6,13 +6,20 @@ import { ButtonSize } from "./createButton";
...
   internalSpacing: BentoSprinkles["gap"];
   iconSize: SizeConfig<IconProps["size"]>;
+  /**
+   * @default true
+   */
   uppercaseLabel: boolean;
+  /**
+   * @default "medium"
+   */
   defaultSize: ButtonSize;
```
Comment (gabro) [inline packages/bento-design-system/src/Button/Config.ts:24]: just samples for now

File: packages/bento-design-system/src/index.ts [hunk 2]
```diff
@@ -89,3 +89,5 @@ export { defaultConfigs };
 export { createBentoComponents } from "./createBentoComponents";
 
 export { createUseComponentsShowcase } from "./createUseComponentsShowcase";
+
+export type { ClassValue } from "clsx";
```
Comment (gabro) [inline packages/bento-design-system/src/index.ts:93]: This fixes some TS complains about "non-portability" of Layout components

File: package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   },
   "scripts": {
     "build": "turbo run build",
-    "start": "turbo run start",
+    "start": "turbo run start --filter=!website",
```
Comment (veej) [inline package.json:9]: 🙏

File: packages/bento-design-system/package.json [hunk 4]
```diff
@@ -110,6 +111,7 @@
     "lint-staged": "12.3.7",
     "mini-css-extract-plugin": "2.6.0",
     "playroom": "0.27.9",
+    "postcss": "^8.4.12",
```
Comment (veej) [inline packages/bento-design-system/package.json:114]: still needed?

File: packages/website/docs/Guides/layout.mdx [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+<SpacingScale />
+
+## Stack
+
+`Stack` is a layout component that arranges its children vertically.
+
+<Canvas path="Layout/StackBasic" />
+
+You can control the space between children with the `space` prop, which accepts finite set of values
```
Comment (veej) [inline packages/website/docs/Guides/layout.mdx:39]:
```suggestion
You can control the vertical space between children with the `space` prop, which accepts finite set of values
```

File: packages/website/src/snippets/Layout/ColumnsContentSizedColumn.tsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { Column, Columns, Placeholder } from "..";
+
+export default function ColumnsContentSizedColumn() {
+  return (
+    <Columns space={24}>
+      <Column width="content">
+        <Placeholder height={200} label="I'm a content-sized column" />
+      </Column>
+      <Column>
```
Comment (veej) [inline packages/website/src/snippets/Layout/ColumnsContentSizedColumn.tsx:10]: useless? Or we want it here for documentation on using a Column without props?

File: packages/website/src/snippets/Layout/ColumnsThreeColumn.tsx [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { Column, Columns, Placeholder } from "..";
+
+export default function ColumnsThreeColumn() {
+  return (
+    <Columns space={24}>
+      <Column width="1/5">
+        <Placeholder height={200} label="1/5" />
+      </Column>
+      <Column>
```
Comment (veej) [inline packages/website/src/snippets/Layout/ColumnsThreeColumn.tsx:10]: same here

**Review by veej — APPROVED** [review]
💣

File: packages/bento-design-system/package.json (hunk 4 above)
Comment (gabro) [reply packages/bento-design-system/package.json:114]: mm, probably not, I'll check

File: packages/website/src/snippets/Layout/ColumnsContentSizedColumn.tsx (hunk 6 above)
Comment (gabro) [reply packages/website/src/snippets/Layout/ColumnsContentSizedColumn.tsx:10]: yep, probably those examples were written before the "auto-insertion" of `Column`. Fixing them

**Pull request merged**